src/TraceEvent/Stacks/CallTree.cs

```diff
@@ -603,6 +603,11 @@ public string DisplayName
         /// </summary>
         public CallTree CallTree { get { return m_callTree; } }
 
+        /// <summary>
+        /// This sets the background color of the CallTreeNode. For CallTreeNodeBase it should be the default color.
+        /// </summary>
+        public string State { get { return "0"; } }
```

Comment:
You should not need to modify CallTree at all.    The CallTree is the 'model' and really should not have GUI state associated with it.  

I note that you have a 'State' property on CallTreeView, which the appropriate place for state like this.   I think you can simply not have this.  If you believe you do need it, we should talk more.